### DISCUSSION TITLE
The user interface freezes in hello system 0.7.0 on an acer aspire v3-571g laptop.

### DISCUSSION THREAD

**Discussion by bus84** [helloSystem/hello#340] [Q&A]
The graphical environment freezes when the Internet is connected in hello system 0.7.0 on an Acer Aspire v3-571g laptop. Without a connected network cable, the system works fine, there was no such problem in the hello system 0.5.0 version.It also freezes when connected to a network via wifi.

**Comment by probonopd** [comment]
Can you confirm this is the hardware in your computer?

PCI 14e4:16b5:1025:0647 Broadcom NetLink BCM57785 Gigabit Ethernet PCIe 
PCI 14e4:4359:105b:e04b Broadcom BCM43228 802.11a/b/g/n

**Reply by bus84** [reply]
The hardware of the network adapters on this laptop
Broadcom NetLink BCM57785 PCI-E Gigabit Ethernet Controller	PCI
Atheros AR5BWB222 Wireless Network Adapter	PCI

**Comment by probonopd** [comment]
Possibly related:
* https://github.com/helloSystem/ISO/issues/355
* https://bugs.freebsd.org/bugzilla/show_bug.cgi?id=260881
